pkg/options/server.go

```diff
@@ -29,6 +29,7 @@ func GetDefaultRateLimiterOptions() http.RateLimiterOptions {
 	return http.RateLimiterOptions{
 		RequestLimit: GetDefaultServeOptionInt("SERVER_RATE_REQUEST_LIMIT", 5),
 		WindowLength: GetDefaultServeOptionInt("SERVER_RATE_WINDOW_LENGTH", 10),
+		ExemptedIPs:  GetDefaultServeOptionStringArray("SERVER_RATE_EXEMPTED_IPS", []string{}),
```

Comment:
Could we add a CLI option for this as well? There's an example in the PR where we first added the rate limiter options: https://github.com/Lilypad-Tech/lilypad/pull/419/commits/d65aa0d0b0f3b9cb2e0471c7ecd9315357739fc9.